app/assets/javascripts/discourse/app/templates/user/posts.hbs

```diff
@@ -1,5 +1 @@
-{{#if this.model.canLoadMore}}
-  {{hide-application-footer}}
```

Comment:
We'll need this `hideApplicationFooter` to be reinserted in the new implementation. Having a footer rendered at the same time as infinite-scrolling leads to some very weird UX

Reply:
Ah perfect! I missed that. Thanks!